.github/workflows/build.yml

```diff
@@ -81,8 +81,38 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install dependencies
         run: sudo apt-get install gcc-mingw-w64 binutils-mingw-w64 mingw-w64-tools
+      - name: Install libusb
+        env: 
```

Comment:
There is a trailing space here.

In case you don't know, you can easily visualize whitespace issues with `git show` or `git diff`. They will be marked in red because the default setting for `core.whitespace` is `blank-at-eol,blank-at-eof,space-before-tab`. Another useful one is `git config apply.whitespace fix` to automatically fix them during some git operations.

https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_whitespace

Reply:
Sorry for this. Thanks for the suggestions ,with `git show` I saw them clearly. Before this I tried finding them with my editor, which wasn't ideal. I didn't know git showed them like that. Thanks!